.codacy.yml

```diff
@@ -1,4 +1,5 @@
 exclude_paths:
  - '**/3rdparty/**'
  - '**/engine/js/cvat-core.min.js'
+ - '**/engine/js/unzip_imgs.js'
```

Comment:
What is a reason to exclude the file from scanning?

Reply:
this is bundled and minified version of `cvat-data/src/js/unzip_imgs.js` which included to scanning.